### ISSUE TITLE
Docker image from 'master' branch does not build

### ISSUE THREAD

**Issue by EvgenyM** [opencv/cvat#962]
An error occurs while building the _cvat_:
`docker-compose build` results in:
```
  Downloading https://files.pythonhosted.org/packages/07/ed/58a5157aa484c6aa4e33d4190fa235ce0c4a78010ddf592af4fc257b539f/scikit-image-0.16.2.tar.gz (28.9MB)
    Complete output from command python setup.py egg_info:
    Python 3.5.2 detected.

    scikit-image 0.16+ supports only Python 3.6 and above.

    For Python 2.7, please install the 0.14.x Long Term Support using:

     $ pip install 'scikit-image<0.15'


    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-tu6ohdfu/scikit-image/
You are using pip version 8.1.1, however version 19.3.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
ERROR: Service 'cvat' failed to build: The command '/bin/sh -c pip3 install --no-cache-dir -r /tmp/requirements/${DJANGO_CONFIGURATION}.txt' returned a non-zero code: 1
```

**Comment by nmanovic** [comment]
@EvgenyM , thanks for the report. We will look and release a new version without the problem.

**Comment by bsekachev** [comment]
Fixed with #965 [Release 0.5.2]
